app/graphql/types/query.rb

```diff
@@ -12,6 +12,9 @@ def collection_field(name, type)
       end
     end
 
+    include ::Foreman::Plugin::GraphqlPluginFields
```

Comment:
Is this something we want to have in `app/graphql/types/base_object.rb` so it's available for every query or is this specific to the query field?

Reply:
Does this also apply for migrations?

Reply:
Yes, applies for mutations as well, updated.